src/input-group/input-group-list-addon.tsx

```diff
@@ -173,19 +177,40 @@ export const InputGroupListAddon = <T, V>({
         </>
     );
 
+    const renderSelector = () => {
+        if (readOnly) {
+            return selected ? (
+                <SelectorReadOnly>
```

Comment:
For the read only display, it's text only and I think we should not render a button element at all (i.e. style it to look not clickable and make it not clickable)